apps/meteor/app/models/server/raw/Sessions.ts

```diff
@@ -985,31 +985,13 @@ export class SessionsRaw extends BaseRaw<ISession> {
 		return [
 			{ key: { createdAt: -1 } },
 			{ key: { loginAt: -1 } },
+			{ key: { searchTerm: 1 }, background: true },
```

Comment:
do we need this? since we're using regex for the query it won't be using the index any way.

Reply:
In this case, we use it, because we are searching for just one object in the document. The problem is, when we use the operator `$or`.

Reply:
how about we make it a partial index then? `{ partialFilterExpression: { searchTerm: { $exists: true } }` since it's an optional field? 👀